runtime/onert/backend/cpu/ops/AddLayer.cc

```diff
@@ -40,7 +40,7 @@ void AddLayer::addFloat32()
       nnfw::cker::ProcessBroadcastShapes(getTensorShape(_lhs), getTensorShape(_rhs), &op_params);
   if (need_broadcast)
   {
-    nnfw::cker::BroadcastBinaryArithmeticOp(
+    nnfw::cker::BroadcastBinaryArithmeticOp<float>(
```

Comment:
If you want add `<float>`, could you also add it to `nnfw::cker::BinaryArithmeticOp()` below?

Reply:
Added `float`, `uint8_t`, and `int32_t` wherever appropriate. Thank you.